types/smallwei__avue/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for avue 3.0
+// Project: https://github.com/nmxiaowei/avue
+// Definitions by: Lyoko <https://github.com/SoulLyoko>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference lib="dom" />
+/// <reference path="./element-plus.d.ts" />
+/// <reference path="./dic.d.ts" />
+/// <reference path="./form.d.ts" />
+/// <reference path="./crud.d.ts" />
+/// <reference path="./tree.d.ts" />
+/// <reference path="./global.d.ts" />
+
+// import { App } from 'vue';
+
```

Comment:
```suggestion
```
please remove any comments (dead code) that has no meaning in context of the file